### PR TITLE
add chinese localization

### PR DESCRIPTION
**Pull request by ChenLingSBU**
add chinese localization.

### CODE REVIEW

File: zh.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
+l10n.translations.en = {
```
Comment (dmitry-mukhin) [inline zh.js]: `en` -> `zh`

File: zh.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    table: {
+      title: '名称',
+      size: '大小'
+    }
+  }
+};
+
+// Pluralization rules taken from:
+// http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html
+l10n.pluralize.en = function (n) {
```
Comment (dmitry-mukhin) [inline zh.js]: `en` -> `zh`

File: zh.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  return n === 1 ? 'one' : 'other';
```
Comment (dmitry-mukhin) [inline zh.js]: according to [this](http://unicode.org/repos/cldr-tmp/trunk/diff/supplemental/language_plural_rules.html)
it should be

`return 'other';`

**Comment by homm** [comment]
@dmitry-mukhin I'll fix
@ChenLingSBU Thanks!

**Pull request merged**